packages/components/text/src/__tests__/__snapshots__/Text.test.win32.tsx.snap

```diff
@@ -5,9 +5,7 @@ exports[`Text all props 1`] = `
   style={
     Object {
       "color": "pink",
-      "fontFamily": "Segoe UI",
-      "fontSize": 12,
-      "fontWeight": "500",
+      "fontVariant": "bodyStandard",
```

Comment:
I would expect fontVariant to be stripped out in the snapshot

Reply:
Updated the snapshot to list fontFamily, fontSize, and fontWeight in recent commits.